src/vma/event/netlink_event.h

```diff
@@ -129,7 +128,7 @@ class route_nl_event: public netlink_event
 
 	virtual  const std::string to_str() const;
 
-	const netlink_route_info* get_route_info() const { return m_route_info; }
+	netlink_route_info* get_route_info() const { return m_route_info; }
```

Comment:
const is removed since the new added member to netlink_route_info is mutable and returned by reference so it can changed. As result we can't return const object of class netlink_route_info and use get_route_val() after that.